lib/scim_rails/config.rb

```diff
@@ -1,26 +1,32 @@
+# frozen_string_literal: true
+
 module ScimRails
   class << self
     def configure
       yield config
     end
 
     def config
-      @_config ||= Config.new
+      @config ||= Config.new
     end
   end
 
+  # Class containing configuration of ScimRails
   class Config
-    ALGO_NONE = "none".freeze
+    ALGO_NONE = "none"
 
-    attr_accessor \
+    attr_writer \
```

Comment:
I broke these out as just "writers" because we override the readers below which create kind of confusing situation where we create a reader just to override it lower in the file. Rubocop flagged that situation for me and so I broke those out so we are just creating their writer up here.